packages/2d/src/Hooks/useInspectorHoverOutline.ts

```diff
@@ -1,30 +1,61 @@
-import { useInspectorHover } from "@hex-engine/inspector";
```

Comment:
I think the changes you made to this file are making this hook too complicated, and so things are getting tangled and confusing; instead of putting logic for outline-due-to-hovering-over-inspector and outline-due-to-hovering-over-entity-while-in-select-mode in the same place, I think they should be separate. You should be able to implement outline-due-to-hovering-over-entity-while-in-select-mode without touching this hook at all.

Reply:
Yep, you're right. I've split the two hooks again, I've also gave the hovering-when-selecting-mode a different style (purple see through overlay with purple outline).